library/src/main/java/com/chuckerteam/chucker/internal/support/FormatUtils.kt

```diff
@@ -0,0 +1,178 @@
+        } catch (e: TransformerException) {
+            xml
+        }
+    }
+
+    @JvmStatic
+    fun getShareText(context: Context, transaction: HttpTransaction): String {
+        var text = ""
+        text += context.getString(R.string.chucker_url) + ": " + transaction.url + "\n"
+        text += context.getString(R.string.chucker_method) + ": " + transaction.method + "\n"
+        text += context.getString(R.string.chucker_protocol) + ": " + transaction.protocol + "\n"
+        text += context.getString(R.string.chucker_status) + ": " +
+            transaction.status.toString() + "\n"
+        text += context.getString(R.string.chucker_response) + ": " +
+            transaction.responseSummaryText + "\n"
+        text += context.getString(R.string.chucker_ssl) + ": " +
+            context.getString(if (transaction.isSsl) R.string.chucker_yes else R.string.chucker_no) + "\n"
+        text += "\n"
+        text += context.getString(R.string.chucker_request_time) + ": " +
+            transaction.requestDateString + "\n"
+        text += context.getString(R.string.chucker_response_time) + ": " +
+            transaction.responseDateString + "\n"
+        text += context.getString(R.string.chucker_duration) + ": " +
+            transaction.durationString + "\n"
+        text += "\n"
+        text += context.getString(R.string.chucker_request_size) + ": " +
+            transaction.requestSizeString + "\n"
+        text += context.getString(R.string.chucker_response_size) + ": " +
+            transaction.responseSizeString + "\n"
+        text += context.getString(R.string.chucker_total_size) + ": " +
+            transaction.totalSizeString + "\n"
+        text += "\n"
+        text += "---------- " + context.getString(R.string.chucker_request) + " ----------\n\n"
+
+        var headers = formatHeaders(transaction.getParsedRequestHeaders(), false)
+
+        if (!TextUtils.isEmpty(headers)) {
+            text += headers + "\n"
+        }
+
+        text += if (transaction.isRequestBodyPlainText) {
+            transaction.getFormattedRequestBody()
+        } else {
+            context.getString(R.string.chucker_body_omitted)
+        }
+
+        text += "\n\n"
+        text += "---------- " + context.getString(R.string.chucker_response) + " ----------\n\n"
+
+        headers = formatHeaders(transaction.getParsedResponseHeaders(), false)
+
+        if (!TextUtils.isEmpty(headers)) {
+            text += headers + "\n"
+        }
+
+        text += if (transaction.isResponseBodyPlainText) {
+            transaction.getFormattedResponseBody()
+        } else {
+            context.getString(R.string.chucker_body_omitted)
+        }
+
+        return text
+    }
+
+    @JvmStatic
+    fun getShareCurlCommand(transaction: HttpTransaction): String {
+        var compressed = false
+        var curlCmd = "curl"
+        curlCmd += " -X " + transaction.method
+        val headers = transaction.getParsedRequestHeaders()
```

Comment:
Can be done on one line, `var curlCmd = "curl -X ${transaction.method}`

Reply:
@redwarp comment was not addressed

Reply:
Addressed it.